CERN/TIGRE
# Remove gpu.lib from mex_CUDA_win64_MVS2015.xml

Tested on:

| Software        | Version           | 
| ------------- |:-------------:|
|**Windows**| 10 |
|**MATLAB**|  2016a |
|**CUDA**| 10.1|
|**Visual Studio**| 2015|
|**GPU**| GTX1060 |

| Software        | Version           | 
| ------------- |:-------------:|
|**Windows**| 10 |
|**MATLAB**|  2016a |
|**CUDA**| 10.1|
|**Visual Studio**| 2015|
|**GPU**| GTX2080Ti |


Steps:
1. Rename mex_CUDA_win64_MVS2015.xml to mex_CUDA_win64.xml
2. Run Compile.m
3. Run InitTIGRE
4. Confirm that it is able to execute the script at TIGRE/MATLAB/Demos/d03_generateData.m without errors.
5. Confirm that it is able to execute the script at TIGRE/MATLAB/Demos/d04_SimpleReconstruction.m without errors.
